vaadin-testbench-core/src/main/java/com/vaadin/karibu/Karibu5Test.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2020 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.karibu;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+/**
+ * Base JUnit 5 class for UI unit tests.
+ * 
+ * Subclasses should typically restrict classpath scanning to a specific package
+ * for faster bootstrap by overriding {@link #scanPackage()} method.
```

Comment:
Should we add a note here that if the extending class overrides the init/clean methods the '`@BeforeEach` annotation needs to be added there as the superclass annotation is not inherited in junit5 if the method is overridden.